src/model/ZoneHVACTerminalUnitVariableRefrigerantFlow.hpp

```diff
@@ -54,11 +56,16 @@ namespace model {
   {
 
    public:
-    explicit ZoneHVACTerminalUnitVariableRefrigerantFlow(const Model& model);
+    explicit ZoneHVACTerminalUnitVariableRefrigerantFlow(const Model& model, bool isFluidTemperatureControl = false);
```

Comment:
New convenience ctor. When isFLuidTemperature Control, it uses CoilHeating/CoolingDXVariableRefrigerantFluidTemperatureControl coils + a FanSystemModel

Reply:
Wait, we want this to be "+ a FanVariableVolume", no?

Reply:
I checked the idd, and you're right. It can be FanSystemModel or FanVariableVolume.